tests/test_ERC721_Mintable.py

```diff
@@ -763,6 +763,33 @@ async def test_safeTransferFrom_to_unsupported_contract(erc721_factory):
         assert error['code'] == StarknetErrorCode.ENTRY_POINT_NOT_FOUND_IN_CONTRACT
 
 
+@pytest.mark.asyncio
+async def test_safeTransferFrom_to_eoa(erc721_factory):
```

Comment:
there's no EOAs here
```suggestion
async def test_safeTransferFrom_to_account(erc721_factory):
```